src/examples/Cluster/ClusterTools/ClusterToolsExample.Node/App.config

```diff
@@ -49,4 +49,19 @@
       ]]>
     </hocon>
   </akka>
-<runtime><assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1" /></runtime></configuration>
\ No newline at end of file
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1" />
+  </runtime>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1" >
+      <dependentAssembly>
+        <assemblyIdentity name="Google.ProtocolBuffers.Serialization" publicKeyToken="55f7125234beb589" culture="neutral" />
```

Comment:
> result of preferring NuGet over Paket ;)

LOL

Should we upgrade to the latest version of protobuffs then? Could you create a Github issue for that @Horusiath ?